src/include/binder/binder_context.h

```diff
@@ -14,6 +14,7 @@
 #include <string>
 
 #include "type/types.h"
+#include "concurrency/transaction.h"
```

Comment:
I need to pass txn in AddTable because in this function, getTableWithName needs txn. And this is the most front level I can find to declare the concurrency::Transaction.